src/app/WriteClient.cpp

```diff
@@ -169,6 +169,7 @@ CHIP_ERROR WriteClient::StartNewMessage()
     if (mState == State::AddAttribute)
     {
         ReturnErrorOnFailure(FinalizeMessage(true));
+        mIsWriteRequestChunked = true;
     }
 
     // Do not allow timed request with chunks.
```

Comment:
Shoudl the "do not allowed timed chunk" just be before the above that changes the mIsWriteRequestChunked?